frame/nomination-pools/src/mock.rs

```diff
@@ -55,16 +55,30 @@ impl sp_staking::StakingInterface for StakingMock {
 		StakingMinBond::get()
 	}
 
-	fn desired_validator_count() -> u32 {
+	fn stash_by_ctrl(_controller: &Self::AccountId) -> Result<Self::AccountId, DispatchError> {
```

Comment:
nit: diff could be made better by avoiding re-orderings.

Reply:
I fixed the reordering to realise why I did that in the first place.
My IDE nags that there is a`Different impl member order from the trait `. :D